src/Elliptic/Systems/Elasticity/Equations.hpp

```diff
@@ -106,43 +106,35 @@ struct Fluxes {
                  domain::Tags::Coordinates<Dim, Frame::Inertial>>;
   using volume_tags = tmpl::list<Tags::ConstitutiveRelationBase>;
   static void apply(
-      const gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
+      gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
```

Comment:
Question for my education:  The first argument in this declaration is not 'const'.   However, in equatons.cpp, the first argument of the apply's *is* const.   Shouldn't this fail to compile?